decidim/decidim
# Add order by in linked_participatory_space_resources

#### :tophat: What? Why?
THe following PR adds the ordering in linked_participatory_space_resources method, which will force the linked objects to be sorted by weight. 

#### :pushpin: Related Issues
*Link your PR to an issue*
- Fixes #9784

#### Testing
1. Go to assemblies  in main website 
2. Observe the order
3. Visit Admin / Assemblies 
4. Change the order parameter 
5. Repeat steps 1 & 2 

### :camera: Screenshots
*Please add screenshots of the changes you're proposing*
![Description](URL)

:hearts: Thank you!
